hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NodeEndpoint.java

```diff
@@ -108,6 +115,10 @@ public Response getDatanodes() {
           if (datanode.getUuid().equals(pipeline.getLeaderId())) {
             leaderCount.getAndIncrement();
           }
+          int openContainerPerPipeline =
+                  reconContainerManager.getPipelineToOpenContainer()
+                  .getOrDefault(pipelineID, 0);
```

Comment:
What is the rationale behind defaulting to pipeline ID `0` here? Wouldn't `get(pipelineID)` be better?